src/services/dashboards/widgets/aws-personal-health-dashboard/AwsPersonalHealthDashboardWidget.vue

```diff
@@ -1,17 +1,193 @@
+                        <div class="popover-title">
+                            {{ $t('Affected resources') }}
+                        </div>
+                        <p-divider class="divider" />
+                        <div v-for="item in value"
+                             :key="item"
+                             class="popover-item"
+                        >
+                            <span>{{ item }}</span>
+                        </div>
+                    </div>
+                </template>
+            </widget-data-table>
+        </div>
+    </widget-frame>
 </template>
 
 <script setup lang="ts">
+import type { ComputedRef } from 'vue';
 import {
-    defineProps,
+    computed, defineExpose,
+    defineProps, reactive, toRefs,
 } from 'vue';
 
 
-import type { WidgetProps } from '@/services/dashboards/widgets/_configs/config';
+import { PBalloonTab, PDivider } from '@spaceone/design-system';
+
+import { store } from '@/store';
+import { i18n } from '@/translations';
+
+import type { Field } from '@/services/dashboards/widgets/_components/type';
+import WidgetDataTable from '@/services/dashboards/widgets/_components/WidgetDataTable.vue';
+import WidgetFrame from '@/services/dashboards/widgets/_components/WidgetFrame.vue';
+import type { WidgetProps, WidgetExpose } from '@/services/dashboards/widgets/_configs/config';
+import { useWidgetFrameProps } from '@/services/dashboards/widgets/_hooks/use-widget-frame-props';
+// eslint-disable-next-line import/no-cycle
+import { useWidgetState } from '@/services/dashboards/widgets/_hooks/use-widget-state';
+
+interface Data {
+    event: string;
+    region_code: string;
+    start_time: string;
+    last_update_time: string;
+    affected_projects: string[];
+}
 
-// eslint-disable-next-line @typescript-eslint/no-unused-vars
 const props = defineProps<WidgetProps>();
 
+const storeState = reactive({
+    regionItems: computed(() => store.state.reference.region.items),
+});
+
+const state = reactive({
+    ...toRefs(useWidgetState<Data[]>(props)),
+    tabs: [
+        { name: 'open_issues', label: i18n.t('Open issues') }, // song-lang
+        { name: 'scheduled_changes', label: i18n.t('Scheduled changes') }, // song-lang
+        { name: 'other_notifications', label: i18n.t('Other notifications') }, // song-lang
+    ],
+    activeTab: 'open_issues',
+    tableFields: computed<Field[]>(() => [ // song-lang
+        { label: 'Event', name: 'event' },
+        { label: 'Region', name: 'region_code' },
+        { label: 'Start time', name: 'start_time' },
+        { label: 'Last update time', name: 'last_update_time' },
+        {
+            label: 'Affected projects',
+            name: 'affected_projects',
+            detailOptions: {
+                enabled: true,
+                type: 'popover',
+            },
+        },
+    ]),
+    tableItems: computed<Data[]>(() => state.data?.map((d) => ({
+        ...d,
+        region_code: storeState.regionItems[d.region_code]?.name,
```

Comment:
nitpick: If you set reference textOptions, you don't need to convert region name.